config/environments/development.rb

```diff
@@ -1,38 +1,30 @@
-# frozen_string_literal: true
-
-require 'active_support/core_ext/integer/time'
+require "active_support/core_ext/integer/time"
 
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
-  # Raise when request includes unpermitted params to help find errors in
-  # dev/test (default value is `nil`, which is used in production)
-  config.action_controller.action_on_unpermitted_parameters = :raise
```

Comment:
I suspect we turned this on earlier to help find bugs in dev. Do we not need it any longer?